test/e2e_flags.go

```diff
@@ -64,5 +65,8 @@ func initializeServingFlags() *ServingEnvironmentFlags {
 	flag.BoolVar(&f.EnableBetaFeatures, "enable-beta", false,
 		"Set this flag to run tests against beta features")
 
+	flag.BoolVar(&f.EnableLogStream, "enable-logstream", true,
+		"Set this flag to stream logs from system components")
```

Comment:
Kind of a nit, but since the default is true the description ends up wrong. I'm wondering if "DisableLogStream" might be better here, which would also make the invocation a bit nicer "-disable-logstream" instead of "-enable-logstream=false". Don't feel very strongly about this if anyone disagrees, though.

Reply:
I don't have strong opinion either. I just used "Enable" to align it with the other flags. Also, the usage of the flag would be more complicated:
```
if ! test.ServingFlags.DisableLogStream {
```

Reply:
Right, though the other flags default false..

Reply:
Pushed.